conf/schema.xml

```diff
@@ -421,10 +421,10 @@
 <!-- Select fields added to on field for Solr suggest. -->
 <copyField source="bpl_mods_abstract_all_types_ms" dest="suggester_description"/>
 <copyField source="mods_tableOfContents_s" dest="suggester_summary"/>
-<copyField source="mods_subject_topic_ms" dest="suggester_subjects"/>
+<copyField source="mods_subject_topic_s" dest="suggester_subjects"/>
```

Comment:
I recall when last I worked on the suggesters, using the single valued _s fields gave me hell when there were actually multiple values.
Take “mods_subject_topic_s” for example. The field could have the value “Object 1" but “mods_subject_topic_ms” could have [“Item 1", “Object 1”]. As the field mods_subject_topic_s would just hold the last indexed data

Reply:
The solution here is going to be identical to similar issues found for compressed search facets. Copying both the _s and _ms fields yields a field that contains all applicable values.